assignment/assignment3/Lambdas/TopSalariedEmployee.java

```diff
@@ -0,0 +1,18 @@
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+public class TopSalariedEmployee {
+    private final List<Employee> list = new ArrayList<>();
+    {
+        list.add(new Employee("CSE", 50000));
+        list.add(new Employee("IT", 20000));
+        list.add(new Employee("EEE", 40000));
+        list.add(new Employee("CSE", 30000));
+        list.add(new Employee("IT", 60000));
+    }
```

Comment:
Please make sure the two lines of space has to be provide here. Please kindly check once you merged before to orgin.